injected/integration-test/test-pages/favicon/config/favicon-disabled.json

```diff
@@ -3,6 +3,10 @@
     "favicon": {
       "state": "disabled",
       "exceptions": []
+    },
+    "themeColor": {
```

Comment:
I was going to ask why this needs to be added in all other tests but then I found this https://github.com/duckduckgo/content-scope-scripts/pull/1561/files#r2007736395 . Let's follow up on this with @shakyShane when he's back as this means new features such as this will have ever larger PRs.